solutions/src/main/java/ru/sberbank/school/task08/state/MapState.java

```diff
@@ -44,4 +57,4 @@ public int hashCode() {
```

Comment:
Abbreviation in name &#39;PRIME&#39; must contain no more than &#39;2&#39; consecutive capital letters.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a719999007>*